packages/app/src/cli/commands/app/versions/list.ts

```diff
@@ -0,0 +1,35 @@
+import {appFlags} from '../../../flags.js'
+import Command from '../../../utilities/app-command.js'
+import versionList from '../../../services/versions-list.js'
+import {loadExtensionsSpecifications} from '../../../models/extensions/load-specifications.js'
+import {AppInterface} from '../../../models/app/app.js'
+import {load as loadApp} from '../../../models/app/loader.js'
+import {globalFlags} from '@shopify/cli-kit/node/cli'
+import {Args, Flags} from '@oclif/core'
+
+export default class VersionsList extends Command {
+  static description = 'List deployed versions of your app.'
+  static examples = ['<%= config.bin %> <%= command.id %>']
+
+  static flags = {
+    ...globalFlags,
+    ...appFlags,
+    'api-key': Flags.string({
+      hidden: false,
+      description: "Application's API key that will be exposed at build time.",
+      env: 'SHOPIFY_FLAG_API_KEY',
+    }),
```

Comment:
I miss the `reset` flag. I have a cached spin running and I'm not able to reset it

Reply:
talked about this offline, `versions list` shouldn't reset your deploy config